week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,7 @@
-// Declare your function first
+function add(a, b) {
+  return a + b;
+}
 
-// Call the function and assign to a variable `sum`
+var sum = add((a = 13), (b = 124));
```

Comment:
Same here: your function definition on lines 1-3 is perfect, however line 5 is incorrect. When you call a function, you do not need to say what value corresponds to which parameter: javascript just follows the order that you provide your arguments in and knows which one is which 🔮